.github/workflows/reusable-build-on-windows.yml

```diff
@@ -99,3 +99,10 @@ jobs:
         run: |
           mv build\\WasmEdge-${{ env.product_version }}-windows.msi WasmEdge-${{ env.product_version }}-windows.msi
           gh release upload ${{ inputs.version }} WasmEdge-${{ env.product_version }}-windows.msi --clobber
+      - name: Run check_symbols.sh
+        run: |
+          chmod +x ./scripts/check_symbols.sh
+          bash ./scripts/check_symbols.sh
```

Comment:
I don't know if `bash` on windows works differently. I thought we don't need `chmod +x` if we execute it with `bash` here?